runtime-kernel/linux-kernel-starfive-jh7110/spec

```diff
@@ -0,0 +1,7 @@
+KERNEL_VER=6.6.20
```

Comment:
Replace `KERNEL_VER` with `UPSTREAM_VER`.

Reply:
Also, how do we know that it's 6.6.20?

Reply:
You can find out the kernel version from the [Makefile](https://github.com/starfive-tech/linux/blob/JH7110_VF2_6.6_v5.12.0/Makefile) in their official repository.

Reply:
Then I would add a comment explaining it in `spec`, right above `UPSTREAM_VER`.

Reply:
~The kernel version is written in the source repository's Makefile. Isn't this supposed to be common knowledge?~
I've added a comment, and you can check if it meets your requirement.